### DISCUSSION TITLE
[Support]: v0.14 removes UI access to still images yet they are used in frigate+ & saved in folder

### DISCUSSION THREAD

**Discussion by dopeytree** [blakeblackshear/frigate#13112] [General Support] [answered]
### Describe the problem you are having

Occasionally want to download the still image associated with a detection.

Reasons.
It's pretty.
It's something I want to keep.

### Steps to reproduce

Open any v0.14 instance of frigate and try to download or export an still image.

### Version

0.14

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### Operating system

UNRAID

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

annie c500

**Comment by NickM-27** [comment, marked as answer]
This is not a bug, please see the FAQ https://github.com/blakeblackshear/frigate/discussions/12724

**Reply by dopeytree** [reply]
🤷‍♂️ All the FAQ indicates is that no one can be arsed to add a download button to the frigate+ section of images... 
no ones asking for the video to be all linked up they just wanna download the images without browsing the folder manually.

**Reply by NickM-27** [reply]
You can right click or long press to download the image
